yesod-core/src/Yesod/Core/Types.hs

```diff
@@ -306,6 +308,24 @@ data Content = ContentBuilder !BB.Builder !(Maybe Int) -- ^ The content and opti
              | ContentFile !FilePath !(Maybe FilePart)
              | ContentDontEvaluate !Content
 
+-- | Represents a Content as a String, rendering at most a specified number of
+-- bytes of the content, and annotating it with the remaining length.
+--
+-- @since 1.6.28.0
+contentToTruncatedString :: Content -> I.Int64 -> String
```

Comment:
`Int64` is a reasonable choice here but it's a little nicer to use `Integer` or `Int` here - those are in `Prelude` and don't require imports for end users.

Reply:
Agreed, but `take` in `Data.ByteString.Lazy` actually [requires](https://hackage.haskell.org/package/bytestring-0.12.2.0/docs/Data-ByteString-Lazy.html#g:13) Int64.